python/cudf/cudf/io/parquet.py

```diff
@@ -666,7 +739,15 @@ def to_parquet(
             row_group_size_rows=row_group_size_rows,
             max_page_size_bytes=max_page_size_bytes,
             max_page_size_rows=max_page_size_rows,
-            **kwargs,
+            partitions_info=list(
+                zip(
+                    partition_offsets,
+                    np.roll(partition_offsets, -1) - partition_offsets,
+                )
+            )[:-1]
+            if partition_offsets is not None
+            else partition_offsets,
```

Comment:
Being explicit in the case of `partitions_offsets is None` might help here.
```suggestion
            else None,
```